tools/benchmark/benchmark_codec_jxl.cc

```diff
@@ -266,7 +266,7 @@ class JxlCodec : public ImageCodec {
                     ThreadPoolInternal* pool, CodecInOut* io,
                     jpegxl::tools::SpeedStats* speed_stats) override {
     io->frames.clear();
-    if (dparams_ != DecompressParams{}) {
+    if (dparams_.max_passes < 100 || dparams_.max_downsampling != 1) {
```

Comment:
Optionally, we wouldn’t have to know what the defaults are if we wrote this as:

```suggestion
    if (dparams_.max_passes != DecompressParams().max_passes || dparams_.max_downsampling != DecompressParams().max_downsampling) {
```

Reply:
Yes, that's nicer than putting numeric values there. Thanks for the suggestion!